packages/panna-sdk/src/react/utils/utils.ts

```diff
@@ -39,3 +39,21 @@ export function getEnvironmentChain() {
 export function getAAChain(testingStatus?: boolean | undefined) {
   return getChain(testingStatus);
 }
+
+/**
+ * Detect user's country based on browser locale.
+ * @returns The detected country code or null if detection fails
+ */
+export function detectUserCountry(): string | null {
+  try {
+    // Get browser locale (e.g., "en-US", "fr-FR")
+    const locale = navigator.language || navigator.languages?.[0];
+    if (locale && locale.includes('-')) {
+      const countryCode = locale.split('-')[1]?.toUpperCase();
```

Comment:
Could there ever be a scenario where the country could have the 3 letter format?

Reply:
Please clarify.

Reply:
yeah i already fixed it. please review the changes to figure it out